app/Resources/translations/default/ModulesWelcomeAdmin.xlf

```diff
@@ -310,6 +310,11 @@
         <target>Sell your first product quicker than you would have wished with our nice onboarding process.</target>
```

Comment:
Something weird. I would expect that it removes the old wordings 🤔

Reply:
The old wording are never removed by our tool for backward compatibility
But since this one was new anyway we can remove it manually